arch/ceva/src/Makefile

```diff
@@ -41,22 +41,13 @@ ifeq ($(CONFIG_WINDOWS_NATIVE),y)
   CFLAGS += -I$(TOPDIR)\sched
 else
   ARCH_SRCDIR = $(TOPDIR)/arch/$(CONFIG_ARCH)/src
-ifeq ($(WINTOOL),y)
-  NUTTX = "${shell cygpath -w $(OUTDIR)/nuttx$(EXEEXT)}"
-  CFLAGS += -I "${shell cygpath -w $(ARCH_SRCDIR)/chip}"
-  CFLAGS += -I "${shell cygpath -w $(ARCH_SRCDIR)/common}"
-  CFLAGS += -I "${shell cygpath -w $(ARCH_SRCDIR)/$(ARCH_SUBDIR)}"
-  CFLAGS += -I "${shell cygpath -w $(ARCH_SRCDIR)/$(CONFIG_ARCH_CHIP)}"
-  CFLAGS += -I "${shell cygpath -w $(TOPDIR)/sched}"
-  LDSCRIPT := ${shell cygpath -m $(LDSCRIPT)}
-else
-  NUTTX = "$(OUTDIR)/nuttx$(EXEEXT)"
-  CFLAGS += -I$(ARCH_SRCDIR)/chip
-  CFLAGS += -I$(ARCH_SRCDIR)/common
-  CFLAGS += -I$(ARCH_SRCDIR)/$(ARCH_SUBDIR)
-  CFLAGS += -I$(ARCH_SRCDIR)/$(CONFIG_ARCH_CHIP)
-  CFLAGS += -I$(TOPDIR)/sched
-endif
+  NUTTX = $(call CONVERT_PATH,$(OUTDIR)/nuttx$(EXEEXT))
+  CFLAGS += -I $(call CONVERT_PATH,$(ARCH_SRCDIR)/chip)
+  CFLAGS += -I $(call CONVERT_PATH,$(ARCH_SRCDIR)/common)
+  CFLAGS += -I $(call CONVERT_PATH,$(ARCH_SRCDIR)/$(ARCH_SUBDIR))
+  CFLAGS += -I $(call CONVERT_PATH,$(ARCH_SRCDIR)/$(CONFIG_ARCH_CHIP))
+  CFLAGS += -I $(call CONVERT_PATH,$(TOPDIR)/sched)
+  LDSCRIPT := $(call CONVERT_PATH,$(LDSCRIPT))
```

Comment:
Should this be
```suggestion
  ifeq ($(WINTOOL),y)
    LDSCRIPT := $(call CONVERT_PATH,$(LDSCRIPT))
  endif
```

Reply:
Why?

Reply:
If it does not give any problems that we can leave it as is. I left a comment because a similar line was missing in the `else` case of the original code.

Reply:
Examined the code again. I think there should not be any issue by having this enabled in both cases.